.github/workflows/e2e.yml

```diff
@@ -0,0 +1,23 @@
+name: End to end tests
+
+on:
+  push
```

Comment:
- [ ] up for discussion, we probably want something more sensible here

Reply:
Maybe this only runs when we merge to develop? Merging to develop suggests you have something ready to be completed?

Reply:
That's what I'm thinking as well: on push / merge to `develop` and when we tag something.